tweakwcs/tests/test_linearfit.py

```diff
@@ -58,6 +59,12 @@ def ideal_large_data(request):
         proper = True
         transform = 'shift'
 
+    elif request.param == 'rshift':
+        angle = (angle, angle)
+        scale = 1
+        proper = True
+        transform = 'rshift'
```

Comment:
How does this get parametrized?  It looks like `fitgeom` doesn't get parametrized anywhere.

Reply:
Try to add `print(f"\nfitgeom: {fitgeom}")` for example after line 304 and see for yourself. I looks like it does get parametrized.

Reply:
You're right, it does.  I just don't understand it!  😅

Reply:
Ok, I think I get it now.  This helped:

https://stackoverflow.com/a/52611167

I feel like I've used this pattern before, but I didn't recall how it worked.

Reply:
Me neither! You are not alone :)

I think it is somewhat similar to https://docs.pytest.org/en/stable/example/parametrize.html#indirect-parametrization-of-optional-implementations-imports